examples/default/main.tf

```diff
@@ -48,17 +44,97 @@ resource "azurerm_resource_group" "this" {
   name     = module.naming.resource_group.name_unique
 }
 
+# Network resources for private endpoint example
+resource "azurerm_virtual_network" "pe" {
+  location            = azurerm_resource_group.this.location
+  name                = "${module.naming.virtual_network.name_unique}-pe"
+  resource_group_name = azurerm_resource_group.this.name
+  address_space       = ["10.10.0.0/16"]
+}
+
+resource "azurerm_subnet" "pe" {
+  address_prefixes     = ["10.10.0.0/24"]
+  name                 = "pe-subnet"
+  resource_group_name  = azurerm_resource_group.this.name
+  virtual_network_name = azurerm_virtual_network.pe.name
+}
+
 # This is the module call
 # Do not specify location here due to the randomization above.
 # Leaving location as `null` will cause the module to use the resource group location
 # with a data source.
 module "test" {
   source = "../../"
 
+  administrator_login          = "mongoAdminUser"    # example only
+  administrator_login_password = "ChangeM3Now!12345" # example only; DO NOT use hard-coded secrets in production
```

Comment:
replace with randomly generated password since hardcoded example password pose a security risk when users deploy this example